test/functional/transactions/0.tx.js

```diff
@@ -476,33 +468,44 @@ describe('POST /api/transactions (type 0)', function () {
 			});
 
 			it('using string with bad format should fail', function (done) {
-				transaction.recipientId = '15738697512051092602'; //Address without L invalid
+				transaction.recipientId = '15738697512051092602.L';
+
+				sendTransaction(transaction, function (err, res) {
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate transaction schema: Object didn\'t pass validation for format address: '+transaction.recipientId);
+					badTransactions.push(transaction);
+					done();
+				}, true);
+			});
+
+			it('using address without letter L at the end should fail', function (done) {
+				transaction.recipientId = '15738697512051092602';
 
 				sendTransaction(transaction, function (err, res) {
 					node.expect(res).to.have.property('success').to.not.be.ok;
-					node.expect(res).to.have.property('message').to.match(/^Invalid transaction body - Failed to validate transaction schema: Object didn't pass validation for format address: /);
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate transaction schema: Object didn\'t pass validation for format address: '+transaction.recipientId);
 					badTransactions.push(transaction);
 					done();
 				}, true);
 			});
 
-			it('using string smaller than expected should fail', function (done) {
+			it('using string shorter than expected should fail', function (done) {
 				transaction.recipientId = '';
 
 				sendTransaction(transaction, function (err, res) {
 					node.expect(res).to.have.property('success').to.not.be.ok;
-					node.expect(res).to.have.property('message').to.match(/^Invalid transaction body - Failed to validate transaction schema: String is too short /);
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate transaction schema: String is too short (0 chars), minimum 1');
 					badTransactions.push(transaction);
 					done();
 				}, true);
 			});
 
-			it('using string with well format but bigger than expected should fail', function (done) {
-				transaction.recipientId = Array(22+1).join('1')+'L'; //more characters than allowed
+			it('using string with good format but larger than expected should fail', function (done) {
+				transaction.recipientId = Array(22+1).join('1')+'L';
```

Comment:
It took me a few seconds to understand what was going on here. This would be more semantic:
```
transaction.recipientId = Array(22).fill('1').join('')+'L';
```